torchao/utils.py

```diff
@@ -607,13 +607,29 @@ def _torch_version_at_least(min_version):
 def is_MI300():
     if torch.cuda.is_available() and torch.version.hip:
         mxArchName = ["gfx940", "gfx941", "gfx942"]
-        archName = torch.cuda.get_device_properties().gcnArchName
+        archName = torch.cuda.get_device_properties(0).gcnArchName
         for arch in mxArchName:
             if arch in archName:
                 return True
     return False
 
 
+def is_MI350():
```

Comment:
optional: if there is any public info (spec sheet, etc) we can link to about this hardware from the docblock, I think that would be awesome! same for below.

Reply:
thanks . I've added Supported AMD GPU Models and their LLVM gfx Codes
will add detail documentation when the time is right.